scripts/package-lib

```diff
@@ -192,8 +218,8 @@ join-elements() {
 # $2 - Expected type of the field, 'string', 'array' or 'map'
 # $3 (optional) - Either 'nonempty' or 'optional'
 check-field() {
-    local signature
-    signature="$(declare -p "$1" 2> /dev/null)"
+    # shellcheck disable=SC2155
```

Comment:
(nit: i sometimes see people put a comment about what the lint message being disabled is, like so:)

```suggestion
    # shellcheck disable=SC2155
    # SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
Seems reasonable. Thanks!